### PR TITLE
Release Notes for v0.5.0

### PR DESCRIPTION
**Pull request by trbehera**
Signed-off-by: Behera, Tushar Ranjan <tushar.ranjan.behera@intel.com>

### CODE REVIEW

**Review by Darshini-Parikh93 — DISMISSED** [review]

**Review by v-vasavi — DISMISSED** [review]

**Review by prateekcr — DISMISSED** [review]

File: release-notes-v0.5.0.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
+v0.5.0
+
+### Components
+
+This release contains a reference implementation of [FIDO Device Onboard (FDO) Proposed Standard](https://fidoalliance.org/specs/FDO/fido-device-onboard-v1.0-ps-20210323/).
+
+It includes 4 components:
+  * Protocol Reference Implementation (PRI): [pri-fidoiot](https://github.com/secure-device-onboard/pri-fidoiot) is a JAVA based implementation of all the components specified in the FDO Proposed Standard.
+  * Client SDK: [client-sdk-fidoiot](https://github.com/secure-device-onboard/client-sdk-fidoiot) is a C based implementation for the device component specified in the FDO Proposed Standard. Additionally it supports an implementation of the device that uses TPM infrastructure.
+  * EPID Verificatio Service: [epid-verification-service](https://github.com/secure-device-onboard/epid-verification-service) is a wrapper service written on top of EPID SDK to assist FDO Rendezvous service and FDO Owner service to perform device signature verification for EPID based devices.
```
Comment (johneasterday) [inline release-notes-v0.5.0.md]: typo - Verificatio -> Verification

File: release-notes-v0.5.0.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+**pri-fidoiot**: Packages have been renamed from org.fido.iot.xxx to org.fidoalliance.fdo.xxx.
+
+**pri-fidoiot**: The owner service can now provide a different owner key that can be used as the manufacturer key after resale.
+
+**pri-fidoiot**: Support for Intel OnDie ECDSA based device attestation has been included.
+
+**epid-verification-service**: It exposes following REST end-points.
+  * Endpoints to retrieve EPID resources such as privrl, sigrl, public key
+  * Endpoints to get EPID signature verification proof result
+  * Endpoints to download cryptomaterial tar ball and cryptomaterial signature files
```
Comment (johneasterday) [inline release-notes-v0.5.0.md]: cryptomaterial -> crypto material

File: release-notes-v0.5.0.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
+v0.5.0
+
+### Components
+
+This release contains a reference implementation of [FIDO Device Onboard (FDO) Proposed Standard](https://fidoalliance.org/specs/FDO/fido-device-onboard-v1.0-ps-20210323/).
+
+It includes 4 components:
+  * Protocol Reference Implementation (PRI): [pri-fidoiot](https://github.com/secure-device-onboard/pri-fidoiot) is a JAVA based implementation of all the components specified in the FDO Proposed Standard.
```
Comment (KiranSukhavasi) [inline release-notes-v0.5.0.md:8]: Do we need to mention about 0.5-rel branch here or not required?

File: release-notes-v0.5.0.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+  * Client SDK: [client-sdk-fidoiot](https://github.com/secure-device-onboard/client-sdk-fidoiot) is a C based implementation for the device component specified in the FDO Proposed Standard. Additionally it supports an implementation of the device that uses TPM infrastructure.
+  * EPID Verificatio Service: [epid-verification-service](https://github.com/secure-device-onboard/epid-verification-service) is a wrapper service written on top of EPID SDK to assist FDO Rendezvous service and FDO Owner service to perform device signature verification for EPID based devices.
+  * Test: [test-fidoiot](https://github.com/secure-device-onboard/test-fidoiot) implements a test-suite that gets executed as part of continuous integration pipeline.
```
Comment (KiranSukhavasi) [inline release-notes-v0.5.0.md:11]: This "test-fidoiot" link is showing Page not found error. Do i need any permission to access this repo? I assume these tests are in Gitlab. Could we recheck this?

File: release-notes-v0.5.0.md [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+
+**pri-fidoiot**, **client-sdk-fidoiot**: A new ServiceInfo module (fdo_sys) has been implemented. The maximum number of bytes that can be sent in a single ServiceInfo message is configurable.
+
+**pri-fidoiot**, **client-sdk-fidoiot**: Implemented RVBypass feature wherein the TO0 and TO1 protocols are skipped and the device connects to the owner for TO2 protocol using the URLs provided in Rendezvous Directives.
+
+**pri-fidoiot**: Packages have been renamed from org.fido.iot.xxx to org.fidoalliance.fdo.xxx.
+
+**pri-fidoiot**: The owner service can now provide a different owner key that can be used as the manufacturer key after resale.
+
+**pri-fidoiot**: Support for Intel OnDie ECDSA based device attestation has been included.
```
Comment (KiranSukhavasi) [inline release-notes-v0.5.0.md:33]: As we are pushing DAL release to next week, Do we need to mention the support for OnDie in the notes here?

File: release-notes-v0.5.0.md [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+
+**epid-verification-service**: It exposes following REST end-points.
+  * Endpoints to retrieve EPID resources such as privrl, sigrl, public key
+  * Endpoints to get EPID signature verification proof result
+  * Endpoints to download cryptomaterial tar ball and cryptomaterial signature files
+
+**epid-verification-service**: Provides Docker Compose based scripts to deploy the services.
+
+**test-fidoiot**: Provides the TestNG based test-suite that is executed for validating the pull requests raised for `pri-fidoiot` and `client-sdk-fidoiot` repositories.
```
Comment (KiranSukhavasi) [inline release-notes-v0.5.0.md:42]: i couldn't access test-fidoiot. Please let me know if this is applicable!!

File: release-notes-v0.5.0.md [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+
+### Discontinued features
+
+None
+
+### Fixed Issues
+
+None
+
+### Known Issues
```
Comment (KiranSukhavasi) [inline release-notes-v0.5.0.md:58]: Do you want to update any known issues from the outstanding defects? If so, we can review the defects and put few incase...

**Review by KiranSukhavasi — CHANGES_REQUESTED** [review]

File: release-notes-v0.5.0.md [hunk 8]
```diff
@@ -0,0 +1,67 @@
+v0.5.0
+
+### Components
+
+This release contains a reference implementation of [FIDO Device Onboard (FDO) Proposed Standard](https://fidoalliance.org/specs/FDO/fido-device-onboard-v1.0-ps-20210323/).
+
+It includes 4 components:
+  * Protocol Reference Implementation (PRI): [pri-fidoiot](https://github.com/secure-device-onboard/pri-fidoiot) is a JAVA based implementation of all the components specified in the FDO Proposed Standard.
+  * Client SDK: [client-sdk-fidoiot](https://github.com/secure-device-onboard/client-sdk-fidoiot) is a C based implementation for the device component specified in the FDO Proposed Standard. Additionally it supports an implementation of the device that uses TPM infrastructure.
```
Comment (DukeDavis12) [inline release-notes-v0.5.0.md]: Additionally --> Additionally,

File: release-notes-v0.5.0.md [hunk 9]
```diff
@@ -0,0 +1,67 @@
...
+
+**pri-fidoiot**, **client-sdk-fidoiot**: Implemented RVBypass feature wherein the TO0 and TO1 protocols are skipped and the device connects to the owner for TO2 protocol using the URLs provided in Rendezvous Directives.
+
+**pri-fidoiot**: Packages have been renamed from org.fido.iot.xxx to org.fidoalliance.fdo.xxx.
+
+**pri-fidoiot**: The owner service can now provide a different owner key that can be used as the manufacturer key after resale.
+
+**pri-fidoiot**: Support for Intel OnDie ECDSA based device attestation has been included.
+
+**epid-verification-service**: It exposes following REST end-points.
```
Comment (DukeDavis12) [inline release-notes-v0.5.0.md]: exposes following --> exposes the following

File: release-notes-v0.5.0.md (hunk 2 above)
Comment (DukeDavis12) [inline release-notes-v0.5.0.md]: tar ball --> tarball

File: release-notes-v0.5.0.md [hunk 10]
```diff
@@ -0,0 +1,67 @@
...
+
+**epid-verification-service**: Provides Docker Compose based scripts to deploy the services.
+
+**test-fidoiot**: Provides the TestNG based test-suite that is executed for validating the pull requests raised for `pri-fidoiot` and `client-sdk-fidoiot` repositories.
+
+### Changes to existing features
+
+**pri-fidoiot**: Source code has been refactored to separate core protocol implementation and example implementations.
+
+**pri-fidoiot**: TLS support has been enabled for the Docker containers provided within `components-samples\demo`.
```
Comment (DukeDavis12) [inline release-notes-v0.5.0.md:48]: Won't TLS support come in the `New features` section?

File: release-notes-v0.5.0.md (hunk 4 above)
Comment (trbehera) [reply release-notes-v0.5.0.md:11]: test-fidoiot repo would be pushed out to GitHub in a couple of days.

File: release-notes-v0.5.0.md (hunk 5 above)
Comment (trbehera) [reply release-notes-v0.5.0.md:33]: No, that is not part of opensource release.

**Review by DukeDavis12 — APPROVED** [review]

**Review by v-vasavi — APPROVED** [review]

**Review by v-vasavi — APPROVED** [review]

**Review by KiranSukhavasi — APPROVED** [review]

**Review by DukeDavis12 — APPROVED** [review]

**Pull request merged**